cm_sdk/models/__init__.py

```diff
@@ -0,0 +1,266 @@
+from enum import Enum
+import json
+from json import JSONEncoder
+import re
+import logging
+
+__author__ = 'e0d'
+
+
+class StringUtil(object):
```

Comment:
Right, but you get namespacing anyways because they're in a module, and if they're just helper functions there's no reason to put them in a class. The pattern of grouping related functions into a class is really un-idiomatic in Python. The typical pattern is to put them in their own module.